homeassistant/components/todoist/calendar.py

```diff
@@ -120,7 +120,7 @@ async def async_setup_platform(
 
     api = TodoistAPIAsync(token)
     coordinator = TodoistCoordinator(hass, _LOGGER, SCAN_INTERVAL, api)
-    await coordinator.async_config_entry_first_refresh()
+    await coordinator.async_refresh()
```

Comment:
Addressing this comment: https://github.com/home-assistant/core/pull/91808/files/d7976634462f4cd7ae6c301a1352c2f837b8669e#r1173537067

Reply:
After looking at this again, I'm not exactly sure where to put this.  I thought you meant within an `async def async_setup` function in `__init__.py`.  But it's not clear how to get the config for a platform in that function or if platforms should have a function like that.  If you could point me to an example I'd appreciate it, thanks!

Reply:
It would be nice to have this setup to support config entries in the future for sure.  Perhaps after all of the bugs are fixed and it is fully working again I can take a look into that.

Reply:
Yeah,  I am not aware of straight forward way to address this (e.g. i don't know how you'd get platform specific yaml config in the integration setup). Given this fixes the immediate problem of the wrong exception being thrown, i think this is a good fix.  Config entries is the right path forward in future PRs, so that sounds good.